### DISCUSSION TITLE
Different folders for patches

### DISCUSSION THREAD

**Discussion by ghost** [PaperMC/Paper#11530] [Ideas]
### Is your feature request related to a problem?

github doesn't support more than 1000 patches in same folder, paper has over 1000

### Describe the solution you'd like.

move patches into folders like /patches/server/0-500 /patches/server/500-1000 ?

### Describe alternatives you've considered.

Not sure

**Comment by lynxplay** [comment]
Not worth it, we won't have patches like this post hardfork anyway.

**Comment by MiniDigger** [comment]
sp basically, we are working on it
